image/scripts/os2borgerpc_setup.sh

```diff
@@ -32,34 +32,27 @@ useradd user -m -p 12345 -s /bin/bash -U
 chfn -f Borger user
 adduser user nopasswdlogin
 
-# Disable showing shortcuts UI for user Borger
-HOME=/home/user/
-HIDDEN_DIR=/home/.skjult
-mkdir -p "$HOME"/.cache/unity
-touch "$HOME"/.cache/unity/first_run.stamp
-
-mkdir -p "$HIDDEN_DIR"/.cache/unity
-
-cp "$HOME"/.cache/unity/first_run.stamp "$HIDDEN_DIR"/.cache/unity/first_run.stamp
-
 # Make now first boot
-touch /etc/bibos/firstboot
+touch /etc/os2borgerpc/firstboot
 
 # Prepare to run jobs
-mkdir -p /var/lib/bibos/jobs
-chmod -R og-r /var/lib/bibos
+mkdir -p /var/lib/os2borgerpc/jobs
+chmod -R og-r /var/lib/os2borgerpc
+
+# Switch display manager to LightDM.
+DEBIAN_FRONTEND=noninteractive apt -y install lightdm
```

Comment:
I noticed that you've added a snippet that installs `at(1)` if it's missing to a few scripts -- it might be a good idea to defensively put `DEBIAN_FRONTEND=noninteractive` in there as well.

Reply:
In this case, it doesn't seem to be necessary - I don't think `at` is modifying any system configuration that requires user input.

In the case of `lightdm` we _know_ that it will stop and ask the user if we dont set `DEBIAN_FRONTEND=noninteractive`.